wordpress-mobile/WordPress-iOS
# Sentry logging: Log filename instead of absolute path

This is a small tweak to our sentry logging so that we use the filename of a crash location instead of the absolute path (this was resulting in logging the absolute path to the file for whoever happened to be the release manager for a given release). This should help Sentry better group together these crashes.

**To test:**

- To test, I edited the `persistentStoreCoordinator` method in `ContextManager`, and added the following at line 248:

```
error = [NSError errorWithDomain:NSSQLiteErrorDomain code:1 userInfo:nil];
SentryStartupEventAddError(startupEvent, error);
```

- I then inserted a breakpoint in the method modified by this PR, to check the incoming filename was logged correctly.

PR submission checklist:

- [ x I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.
